web/styles/settings.css

```diff
@@ -513,14 +517,23 @@ input[type="checkbox"] {
     vertical-align: top;
     height: auto !important;
     width: auto !important;
+    white-space: break-spaces !important;
 
+    overflow-wrap: anywhere;
     background-color: transparent;
     border-radius: 4px;
     margin-top: 14px;
     margin-left: 10px;
     color: hsl(156, 30%, 50%);
     padding: 3px 10px;
 
+    &.account-alert-notification {
+        margin-bottom: 10px;
+        margin-left: 0;
+        margin-top: 0;
```

Comment:
Can we change this to use single `margin` property?

Reply:
Changed it to a single margin statement and updated the commit messages according to your suggestion.